server/src/main/java/org/elasticsearch/index/IndexMode.java

```diff
@@ -90,13 +90,11 @@ void validateWithOtherSettings(Map<Setting<?>, Object> settings) {
                 }
             }
             settingRequiresTimeSeries(settings, IndexMetadata.INDEX_ROUTING_PATH);
```

Comment:
routing_path also the default problem. e.g:
```
{
  "settings": {
    "index.mode": "time_series",
    "index.routing_path": ""
  }
}
```

it will failed as:
```
{
    "error": {
        "root_cause": [
            {
                "type": "illegal_argument_exception",
                "reason": "[index.mode=time_series] requires [index.routing_path]"
            }
        ],
        "type": "illegal_argument_exception",
        "reason": "[index.mode=time_series] requires [index.routing_path]"
    },
    "status": 400
}
```